src/Enhavo/Bundle/NewsletterBundle/Event/NewsletterEvents.php

```diff
@@ -11,7 +11,11 @@
 
 class NewsletterEvents
 {
-    const EVENT_ADD_SUBSCRIBER = 'newsletter.add_subscriber';
     const EVENT_CREATE_SUBSCRIBER = 'newsletter.create_subscriber';
-    const EVENT_CLEVERREACH_PRE_SEND = 'newsletter.cleverreach_pre_send';
-}
\ No newline at end of file
+    const EVENT_PRE_ADD_SUBSCRIBER = 'newsletter.pre_add_subscriber';
```

Comment:
In newer symfony versions a name is not required for events. It is already deprecated in 4.4. We should use different Event classes for this.

Reply:
Please take a closer look at the implementation.
Does it make sense to have a Event class for every Pre/Post event?
At the moment there is a SubscriberEvent with a Type and is dispatched like so:
`$event = new SubscriberEvent(NewsletterEvents::EVENT_CREATE_SUBSCRIBER, $subscriber);
        $this->eventDispatcher->dispatch(NewsletterEvents::EVENT_CREATE_SUBSCRIBER, $event);`

Reply:
Maybe a event class for every event listed here is not necessary, but i would prefer to move the constants to the event class itself and remove the `NewsletterEvent`. That would make the scope of the "type" more clear